tests/test_hdfs.py

```diff
@@ -0,0 +1,117 @@
+import logging
+import pytest
+import shakedown
+
+import sdk_auth
+import sdk_hosts
+import sdk_install
+
+from tests import utils
+
+
+log = logging.getLogger(__name__)
+GENERIC_HDFS_USER_PRINCIPAL = "hdfs@{realm}".format(realm=sdk_auth.REALM)
+# To do: change when no longer using HDFS stub universe
+HDFS_PACKAGE_NAME='beta-hdfs'
```

Comment:
again maybe warn here? also why not put all of the code relating to `stub-universe` usage in one place?

Reply:
Let me check ...

Reply:
I'll move the adding of the hdfs stub universe into this file. Once HDFS with kerberos is released, we can remove that code.